crates/sncast/src/lib.rs

```diff
@@ -84,6 +85,16 @@ pub enum Network {
     Sepolia,
 }
 
+impl Display for Network {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let str = match self {
+            Network::Mainnet => "mainnet".to_string(),
+            Network::Sepolia => "sepolia".to_string(),
+        };
+        write!(f, "{str}")
+    }
+}
+
```

Comment:
This one should be faster
```suggestion
impl Display for Network {
    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
        match self {
            Network::Mainnet => write!(f, "mainnet"),
            Network::Sepolia => write!(f, "sepolia"),
        }
    }
}

```